library/elasticsearch

```diff
@@ -12,10 +12,10 @@
 1.6.2: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.6
 1.6: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.6
 
-1.7.1: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
-1.7: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
-1: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
-latest: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
+1.7.2: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
+1.7: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
+1: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
+latest: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
```

Comment:
These need to be updated to `18527ddda24e2f765c18c925f8970b5570882a3c` now. :+1:

(As seen in the output of `./generate-stackbrew-library.sh` when run within the `elasticsearch` repo.)